### PR TITLE
Clean up #1249

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
I didn't split this up into atomic commits because there was too much to fix.

### CODE REVIEW

File: source/agora/stats/Stats.d [hunk 1]
```diff
@@ -13,28 +13,36 @@
...
-
+    ///
     public alias ValueType ValueTypeT;
+
+    ///
     public alias LabelType LabelTypeT;
-    public alias StatValueLabelTup = Tuple!(const ValueType, "value", const LabelType, "label");
+
+    ///
+    public struct StatValueLabelTup
```
Comment (Geod24) [inline source/agora/stats/Stats.d]: No need to call it Tup anymore ?

File: source/agora/stats/Stats.d [hunk 2]
```diff
@@ -51,21 +59,20 @@ public struct Stats (ValueType, LabelType)
 
     ***************************************************************************/
 
-    public void increaseMetricBy (string metricName, LabelTs...)(ulong number, LabelTs labels_packed) pure nothrow @safe
+    public void increaseMetricBy (string metricName, LabelTs...)(ulong number,
```
Comment (Geod24) [inline source/agora/stats/Stats.d]: `number` => `amount` ?

**Review by Geod24 — APPROVED** [review]
Nice. Just two comments.

File: source/agora/stats/Stats.d (hunk 1 above)
Comment (Geod24) [reply source/agora/stats/Stats.d]: Nope ?

Comment (AndrejMitrovic) [reply source/agora/stats/Stats.d]: Indeed, forgot to save. :>

**Comment by Geod24** [comment]
The failure on LDC + ubuntu is due to network issues.

**Pull request merged**